sys/include/net/gnrc/netif/hdr.h

```diff
@@ -199,6 +199,35 @@ gnrc_pktsnip_t *gnrc_netif_hdr_build(uint8_t *src, uint8_t src_len, uint8_t *dst
  */
 void gnrc_netif_hdr_print(gnrc_netif_hdr_t *hdr);
 
+/* @brief   Fetch the netif header flags of a gnrc packet
+ *
+ * @param[in]   pkt     gnrc packet from whom to fetch
+ *
+ * @return              netif header flags of @p pkt
+ * @return              0, if no header is present
+ */
+uint8_t gnrc_netif_hdr_get_flag(gnrc_pktsnip_t* pkt);
+
+/* @brief   Extract the destination address out of a gnrc packet
+ *
+ * @param[in]   pkt                 gnrc packet from whom to extract
+ * @param[out]  pointer_to_addr     pointer to address will be stored here
+ *
+ * @return                          length of destination address
+ * @return                          -ENOENT, if there is no destination address field
```

Comment:
An IPv6 header also has an address field ;). So no: this is not inclusionary ;). I would just do it in one sentence.